### PR TITLE
BC-745 Integrate codeowners with coverage

### PR DESCRIPTION
**Pull request by Patreon-Amber**
has #comment: WRT python modules, glob to all the .py files under the
dir, because we have mixed in .html and such that exist underneath a
module, but don't actually have python code owned by it.

### CODE REVIEW

File: hooks/create_codeowners.py [hunk 1]
```diff
@@ -67,7 +69,7 @@ def execute(args):
             ):
                 # if initializer has declaration, assume owner wants ownership over whole package
                 file_path = file_path.replace(
-                    "__init__.py", ""
+                    "__init__.py", "**/*.py"
```
Comment (emosesPatreon) [inline hooks/create_codeowners.py]: Add a test please.

**Comment by Patreon-Amber** [comment]
Added a circle config so it actually runs the tests: https://circleci.com/gh/Patreon/precommit/5?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
And also fixed tests and warnings.

File: hooks/check_codeowners.py [hunk 2]
```diff
@@ -59,25 +59,28 @@ def execute(args):
     for file in args.filenames:
         file_contents = file.read()
```
Comment (Patreon-Amber) [inline hooks/check_codeowners.py]: This left dangling open file handles around.

File: tests/test_create_codeowners.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 from hooks.util import FAIL
 from hooks.util import PASS
 
-regex_pattern = "--regex-pattern=__codeowner__\s*=\s*['\"]([\S\s]+)['\"]"
+regex_pattern = r"--regex-pattern=__codeowner__\s*=\s*['\"]([\S\s]+)['\"]"
```
Comment (Patreon-Amber) [inline tests/test_create_codeowners.py]: `DeprecationWarning: Invalid escape sequence \s`

Let this be a raw string, don't do any interpretation until it goes into the regex.

File: hooks/check_codeowners.py (hunk 2 above)
Comment (teeeg) [reply hooks/check_codeowners.py]: thank you

**Comment by teeeg** [comment]
you added codeowners to each file, did you also want to add create-codeowners and check-codeowners as hooks precommits to this repo?

**Comment by Patreon-Amber** [comment]
> you added codeowners to each file, did you also want to add create-codeowners and check-codeowners as hooks precommits to this repo?

Just trying to get in the habit of adding attribution.  For now I figured best-effort is good enough, it can be forced later if we really think it's necessary.

**Pull request merged**